bundle.mustache

```diff
@@ -10,31 +13,44 @@ URL: {{{PROJECTURL}}}
 {{#each SOURCES}}
 {{{.}}}
 {{/each}}
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}npm
+%else
 BuildRequires: nodejs-packaging
+%endif
 BuildArch: noarch
 ExclusiveArch: %{nodejs_arches} noarch
 
-Provides: npm(%{npm_name}) = %{version}
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
 {{#each PROVIDES}}
 Provides: bundled(npm({{{name}}})) = {{{version}}}
```

Comment:
```suggestion
Provides: bundled(%{?scl_prefix}npm({{{name}}})) = {{{version}}}
```

Reply:
We sure this should include the prefix in the bundled? rh-nodejs10 did not approach it that way.